tests/ensemble_evaluator/test_ensemble_evaluator.py

```diff
@@ -115,14 +112,18 @@ def test_dispatchers_can_connect_and_monitor_can_shut_down_evaluator(evaluator):
     monitor = evaluator.run()
     events = monitor.track()
 
+    ee_config = load_config()
+    host = ee_config.get("host")
+    port = ee_config.get("port")
+
     # first snapshot before any event occurs
     snapshot_event = next(events)
     snapshot = Snapshot(snapshot_event.data)
     assert snapshot.get_status() == "Unknown"
 
     # two dispatchers connect
-    with Client(evaluator._host, evaluator._port, "/dispatch") as dispatch1, Client(
-        evaluator._host, evaluator._port, "/dispatch"
+    with Client(host, port, "/dispatch") as dispatch1, Client(
+        host, port, "/dispatch"
```

Comment:
This test now hangs :confused:

Reply:
Yeah, the reason was that we tried to cancel an ensemble that was not cancellable. Quick fix was to allow evaluator to decide whether or not to cancel, but we need to discuss this.